app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+
+    if len(command_list) < 3:
+        return
+
+    cmd = command_list[0]
```

Comment:
u use cmd only in condition so u don't need to unpack this value, just use if not startwith method.  And of course u can group up your condition using operator  (or)